shimmingtoolbox/cli/b0shim.py

```diff
@@ -70,18 +70,18 @@ def b0shim_cli():
                    "'--slice-factor' value is '3', then with the 'sequential' mode, shimming will be performed "
                    "independently on the following groups: {0,1,2}, {3,4,5}, etc. With the mode 'interleaved', "
                    "it will be: {0,2,4}, {1,3,5}, etc.")
-@click.option('--optimizer-method', 'method', type=click.Choice(['least_squares', 'pseudo_inverse', 'quad_prog']),
-              required=False, default='quad_prog', show_default=True,
-              help="Method used by the optimizer. LS, and QP will respect the constraints,"
-                   "PS will not respect the constraints")
+@click.option('--optimizer-method', 'method', required=False, default='quad_prog', show_default=True,
+              type=click.Choice(['least_squares', 'pseudo_inverse', 'quad_prog', 'gradient']),
+              help="Method used by the optimizer. LS, QP and gradient will respect the constraints,"
```

Comment:
I'm not sure if gradient is is the right name to call this.. L-BFGS is not easy to remember though...

Reply:
Yes, that's why I thought a more general name was better for that.